v2/pkg/runner/options.go

```diff
@@ -314,3 +314,153 @@ func userHomeDir() string {
 	}
 	return usr.HomeDir
 }
+
+// ParseUserOptions parses the command line flags provided by a user
+func ParseUserOptions(options *Options) *Options {
```

Comment:
The `ParseOptions` function is geared toward the CLI and should not be available in the API. To address this, a new function called `Init()` -or something different- can be introduced for handling Options. It will perform all the tasks of `ParseOptions` except the CLI operations. We can use this new function in `ParseOptions`, `runner.NewRunner`, or any other desired location.